cli/mobycli/exec.go

```diff
@@ -63,10 +62,8 @@ func mustDelegateToMoby(ctxType string) bool {
 }
 
 // Exec delegates to com.docker.cli if on moby context
-func Exec(root *cobra.Command) {
-	childExit := make(chan bool)
-	err := RunDocker(childExit, os.Args[1:]...)
-	childExit <- true
+func Exec() {
```

Comment:
If I understood correctly the signature change is just for code cleaning, right?

Reply:
yes, parameter is now unused (lint)